sources/packages/backend/apps/api/src/services/application/application.service.ts

```diff
@@ -621,6 +621,9 @@ export class ApplicationService extends RecordDataModelService<Application> {
         location: {
           id: true,
           name: true,
+          institution: {
+            legalOperatingName: true,
```

Comment:
I would strongly suggest to add id as well while including a relation, as typeorm query doesn't seem to be formed efficiently when the property id is missed. 

![image](https://github.com/user-attachments/assets/bc5e2734-e0f1-4e5f-8d50-e54531ac6053)